packages/coding-standards/ecs.php

```diff
@@ -140,180 +131,174 @@
+    ->withRules([
+        InlineDocCommentDeclarationSniff::class,
+        NullableTypeForNullDefaultValueSniff::class,
+        ReturnTypeHintSpacingSniff::class,
+    ])
+    ->withConfiguredRule(
+        ForbiddenClassesSniff::class, [
         'forbiddenClasses' => [
             'Overblog\GraphQLBundle\Error\UserError' => null,
             'GraphQL\Error\UserError' => null,
         ],
-    ]);
-
-    // Shopsys Checkers
-    $ecsConfig->rule(InheritDocFormatFixer::class);
-    $ecsConfig->rule(ForbiddenDumpFixer::class);
-    $ecsConfig->rule(MissingButtonTypeFixer::class);
-    $ecsConfig->rule(OrmJoinColumnRequireNullableFixer::class);
-    $ecsConfig->rule(RedundantMarkDownTrailingSpacesFixer::class);
-    $ecsConfig->rule(ObjectIsCreatedByFactorySniff::class);
-    $ecsConfig->rule(ForbiddenDumpSniff::class);
-    $ecsConfig->rule(ForbiddenExitSniff::class);
-    $ecsConfig->rule(ForbiddenSuperGlobalSniff::class);
-    $ecsConfig->rule(ForbiddenDoctrineInheritanceSniff::class);
-    $ecsConfig->rule(ForbiddenDoctrineDefaultValueSniff::class);
-    // method arguments and variables should be $camelCase
-    $ecsConfig->rule(ValidVariableNameSniff::class);
-    // add all @param, @return and @var annotations, in FQN
-    $ecsConfig->rule(MissingParamAnnotationsFixer::class);
-    $ecsConfig->rule(MissingReturnAnnotationFixer::class);
-    $ecsConfig->rule(OrderedParamAnnotationsFixer::class);
-    $ecsConfig->rule(FullyQualifiedClassNameInAnnotationSniff::class);
-
-    // Custom Shopsys standards
-    $ecsConfig->rule(EmptyStatementSniff::class);
-    $ecsConfig->rule(ForLoopShouldBeWhileLoopSniff::class);
-    $ecsConfig->rule(ForLoopWithTestFunctionCallSniff::class);
-    $ecsConfig->rule(JumbledIncrementerSniff::class);
-    $ecsConfig->rule(UnconditionalIfStatementSniff::class);
-    $ecsConfig->rule(TodoSniff::class);
-    $ecsConfig->rule(FixmeSniff::class);
-    $ecsConfig->rule(NoSpaceAfterCastSniff::class);
-    $ecsConfig->rule(CallTimePassByReferenceSniff::class);
-    $ecsConfig->ruleWithConfiguration(CyclomaticComplexitySniff::class, [
+    ])
+    ->withRules([
+        // Shopsys Checkers
+        InheritDocFormatFixer::class,
+        ForbiddenDumpFixer::class,
+        MissingButtonTypeFixer::class,
+        OrmJoinColumnRequireNullableFixer::class,
+        ObjectIsCreatedByFactorySniff::class,
+        ForbiddenDumpSniff::class,
+        ForbiddenExitSniff::class,
+        ForbiddenSuperGlobalSniff::class,
+        ForbiddenDoctrineInheritanceSniff::class,
+        ForbiddenDoctrineDefaultValueSniff::class,
+        // method arguments and variables should be $camelCase
+        ValidVariableNameSniff::class,
+        // add all @param, @return and @var annotations, in FQN
+        MissingParamAnnotationsFixer::class,
+        MissingReturnAnnotationFixer::class,
+        OrderedParamAnnotationsFixer::class,
+        FullyQualifiedClassNameInAnnotationSniff::class,
+        // Custom Shopsys standards
```

Comment:
I know it has been there already, however, I am not sure whether I understand this comment and I feel like I want to get rid of it :smile:

Reply:
same as above